src/scripts/uploadPremadeBundles.js

```diff
@@ -100,11 +101,21 @@ async function main() {
     // read each EXM bundle file
     const data = fs.readFileSync(filePath, 'utf8');
     if (data) {
-      console.log(`Uploading ${filePath.split('/').slice(-1)}...`);
       const bundle = JSON.parse(data);
+      if (bundle.resourceType !== 'Bundle') {
+        console.log(`Skipping ${filePath.split('/').slice(-1)} NOT A BUNDLE`);
+        continue;
+      }
+      //console.log(`Uploading ${filePath.split('/').slice(-1)}...`);
+
       // retrieve each resource and insert into database
       const uploads = bundle.entry.map(async res => {
         try {
+          // If there is no ID... make one probably MADiE Bundle Measure
+          if (!res.resource.id) {
+            res.resource.id = filePath.split('/').slice(-1)[0].split('-')[0] || randomUUID();
```

Comment:
Look at using a uuid4 for this instead. It happens in the POST cases on the server.

Reply:
Now using `uuidv4()` similar to other places in codebase.